semeio/jobs/rft/utility.py

```diff
@@ -0,0 +1,265 @@
+import os
+import datetime
+import argparse
+
+from ecl.rft import EclRFTFile
+from ecl.grid import EclGrid
+from semeio.jobs.rft.zonemap import ZoneMap
+
+
+def strip_comments(line):
+    return line.partition("--")[0].rstrip()
+
+
+def existing_directory(path):
+    if not os.path.exists(path):
+        raise argparse.ArgumentTypeError("The path {} does not exist".format(path))
+    return path
+
+
+def load_zonemap(path):
+    if not os.path.isfile(path):
+        raise argparse.ArgumentTypeError(
+            "ZoneMap file {filename} not found!".format(filename=path)
+        )
+
+    zones_at_k_value = load_and_parse_zonemap_file(path)
+
+    return ZoneMap(zones_at_k_value)
+
+
+def parse_trajectory_line(line):
+    point = line.split()
+    if len(point) < 4 or len(point) > 5:
+        raise argparse.ArgumentTypeError(
+            "Trajectory data file not on correct format: 'utm_x utm_y md tvd <zone>' - zone is optional"
+        )
+
+    try:
+        floats = [float(v) for v in point[:4]]
+    except ValueError:
+        raise argparse.ArgumentTypeError(
+            "Error: Failed to extract data from line {line}".format(line=line)
+        )
+
+    zone = point[4].strip() if len(point) == 5 else None
+    return floats + [zone]
+
+
+def load_trajectory_points(filepath):
+    """
+    The file must be on the format of <FLOAT FLOAT FLOAT FLOAT STRING> which
+    represents <well_name day month year report_step>
+
+    Parameters
+    ----------
+    filename : string
+        Filename to open
+
+    Returns
+    -------
+    [Tuple]
+        Returns a list of tuples, where each tuple consists of 4 floats and a string/None
+    """
+
+    trajectory_points = []
+
+    filename = os.path.join(filepath)
+    if not os.path.isfile(filename):
+        raise argparse.ArgumentTypeError(
+            "Warning: Trajectory file {filename} not found!".format(filename=filename)
+        )
+
+    with open(filename, "r") as f:
+        trajectory_lines = f.readlines()
+
+    trajectory_lines = [strip_comments(l) for l in trajectory_lines]
+
+    for line in trajectory_lines:
+        if not line:
```

Comment:
be explicit. `line is None or line is ""` or something along those lines.

Reply:
I see what you did there ;)

Reply:
But the recommended practice from PEP8 is to: 

>For sequences, (strings, lists, tuples), use the fact that empty sequences are false.

```
Yes: if not seq:
     if seq:

No:  if len(seq):
     if not len(seq):
```